wa/workloads/drarm/__init__.py

```diff
@@ -0,0 +1,103 @@
+    install_timeout = 200
+    description = """
+    Dr. Arm’s Amazing Adventures is a “Souls-Like” Mobile Action Role Playing Game developed at Arm.
+    """
+
+    parameters = [
+        Parameter('timeout', kind=int, default=61,
+                  description='The amount of time the game should run for'),
+        Parameter('auto_demo', kind=bool, default=False,
+                  description='Start the demo automatically'),
+        Parameter('show_fps', kind=bool, default=False,
+                  description='Show the FPS count window in-game'),
+        Parameter('adpf', kind=bool, default=True,
+                  description='Enable ADPF'),
+        Parameter('adpf_auto', kind=bool, default=True,
+                  description='Enable automatic ADPF mode'),
+        Parameter('adpf_logging', kind=bool, default=False,
+                  description='Enable ADPF logging'),
+        Parameter('verbose_log', kind=bool, default=False,
+                  description='Emit reported stats as debug logs'),
+        Parameter('adpf_interventions', kind=bool, default=True,
+                  description='Enable ADPF interventions'),
+        Parameter('target_vsyncs', kind=int, default=1,
+                  description='the number of vsyncs to target to a frame (1 = current display rate)'),
+        Parameter('target_framerate', kind=int, default=None,
+                  description='Target framerate for the application'),
+        Parameter('fps_report_file', kind=str, default=None,
+                  description='File name that the ADPF FPS report should use.'),
+        Parameter('fixed_time_step', kind=float, default=None,
+                  description='Time, in seconds, that should be used in advancing the simulation'),
+    ]
+
+    @property
+    def apk_arguments(self):
+        args = {
+            'showFPS': int(self.show_fps),
+            'doAdpf': int(self.adpf),
+            'adpfMode': int(self.adpf_auto),
+            'adpfLogging': int(self.adpf_logging),
+            'verboseLog': int(self.verbose_log),
+            'adpfInterventions': int(self.adpf_interventions),
+            'targetVsyncs': self.target_vsyncs,
+            'autoDemo': int(self.auto_demo),
+        }
+
+        if self.target_framerate is not None:
+            args['targetFramerate'] = self.target_framerate
+
+        if self.fixed_time_step is not None:
+            args['fixedTimeStep'] = self.fixed_time_step
+
+        if self.fps_report_file is not None:
+            args['fpsReportFileName'] = self.fps_report_file
+
+        return args
+
+    def run(self, context):
+        self.target.sleep(self.timeout)
+
+    def update_output(self, context):
+        super(DrArm, self).update_output(context)
+        outfile_glob = self.target.path.join(self.target.persistent_data_path, self.apk.package, 'files', '*.csv')
+
+        try:
+            ls_output = self.target.execute('ls {}'.format(outfile_glob), as_root=True)
+        except Exception:
```

Comment:
Can you be more specific in the exception here. Should this be a `TargetStableError` etc.?

Reply:
Good point, thanks, it should be `TargetStableCalledProcessError`